set.mm

```diff
@@ -29664,6 +29664,19 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     ( cv wcel wi wral pm2.04 ralimi2 impbii ) BEZDFZAGZBCHLCFZAGZBDHNPBCDOMAIJP
     NBDCMOAIJK $.
 
+  ${
+    $d y A $.  $d x B $.  $d x y $.
+    reeanlem.1 $e |- ( E. x E. y ( ( x e. A /\ ph ) /\ ( y e. B /\ ps ) )
+                    <-> ( E. x ( x e. A /\ ph ) /\ E. y ( y e. B /\ ps ) ) ) $.
+    $( Lemma factoring out common proof steps of ~ reeanv and ~ reean .
+       (Contributed by Wolf Lammen, 20-Aug-2023.) $)
```

Comment:
Maybe keep Norm's attribution and add your "proof modified by"?

Reply:
I thought about this, because I am certainly not the 'inventor' of this code, but just extracted portions of an existing proof.  The original proof idea was designed by NM and AS, latest change dating back in 2011, but I don't know who did what, and cannot use git to find out.  So attributing NM alone could be unfair to AS.

This is a general problem in case you refactor a given proof base.  Adding all possible contributors could turn out to be a nightmare, if the base is was designed by, say 10 or more individuals.  Doing the detective work to identify each contribution is not in support of refactoring at all, and might severely hamper development.  So what shall we do?  I am curious to here opinions on this topic.

Reply:
That's fine with me! It's not always easy to manage attributions.